receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -69,6 +69,14 @@ resource_attributes:
     description: "Glusterfs volume path"
     enabled: true
     type: string
+  container.csi.volume.id:
```

Comment:
I'm bit concerned from the semantic conventions perspective since those are not yet defined as resource attributes there. See https://github.com/open-telemetry/semantic-conventions/pull/1337#issuecomment-2421573184

Reply:
Just to avoid possible confusion here. It has been added in SemConv with https://github.com/open-telemetry/semantic-conventions/pull/1337 but is only part of the registry and not defined as Resource or metric attribute. So I'm not sure if we can already use it.

Reply:
I opened https://github.com/open-telemetry/semantic-conventions/pull/1499 to clear this up. Is there anything else you can see I'm missing?